pkg/operator/v410_00_assets/bindata.go

```diff
@@ -1701,7 +1701,7 @@ spec:
       requests:
         memory: 50Mi
         cpu: 10m
-  terminationGracePeriodSeconds: 135 # bit more than 70s (minimal termination period) + 60s (apiserver graceful termination)
+  terminationGracePeriodSeconds: {{.GracefulTerminationDuration}}
```

Comment:
@sttts actually, maybe we should add a default value here?

The config observer controller might be slow and we could install a server with a broken config.

I think there is no guarantee that the target controller and the config controller run in lockstep.

Reply:
This should only matter on first 4.8 upgrade, no? The race has no impact because we immediate roll out another revision. So we have a sub-second race I guess.